schema/crdb/no-default-pool-for-internal-silo/up02.sql

```diff
@@ -0,0 +1,18 @@
+/*
+ * Add check constraint ensuring that, if the pool is linked
+ * to the internal Oxide services silo, it's not marked as
+ * a default pool.
+ */
+ALTER TABLE IF EXISTS
+omicron.public.ip_pool_resource
```

Comment:
As a nitpick, this could be `ALTER TABLE omicron.public.ip_pool_resource` - you really **do** want the table to exist at this point. It's just the constraint that may or may not have been created.

Reply:
Done in ca66077842d81381071b0ad686c862d39931ba96